compiler/lib/linker.ml

```diff
@@ -437,7 +437,7 @@ let list_all ?from () =
     provided
     StringSet.empty
 
-let load_fragment ~target_env ~filename (f : Fragment.t) =
+let load_fragment ~ignore_always_annotation ~target_env ~filename (f : Fragment.t) =
```

Comment:
Can you explain the usecase for ignore always annotations?

Reply:
@vouillon

Reply:
We are currently linking the JavaScript runtime. But we don't want to include the last lines of `runtime/sys.js`

I'm not sure whether this really make sense, though... Maybe we should just link a file with the primitives we care about (`caml_js_on_ie`, `caml_js_html_escape` and `caml_js_html_entities`).

Reply:
Do we really want to enforce this behavior? Another way to do this would be to add a new annotations to the js stubs `//Target: Wasm`.

Reply:
I think we should just remove this change.
See https://github.com/ocaml-wasm/wasm_of_ocaml/pull/72

Reply:
Actually, we need to parse the JavaScript runtime files to get the primitive annotations.
I think I'm going to add a flag `//If: wasm`

Reply:
See #1674